packages/onboarding/styles/mixins.scss

```diff
@@ -157,15 +157,7 @@
 @mixin onboarding-block-edge2edge-container {
 	@include onboarding-block-edge2edge;
 
-	width: calc( 100% + #{$onboarding-block-margin-mobile * 2} );
-
-	@include break-small {
-		width: calc( 100% + #{$onboarding-block-margin-small * 2} );
-	}
-
-	@include break-medium {
-		width: calc( 100% + #{$onboarding-block-margin-medium * 2} );
-	}
+	width: auto;
```

Comment:
It looks like `width: auto` is enough to force the browser to make the element wider (thanks to the negative margins)